Xlib/ext/dpms.py

```diff
@@ -159,10 +162,15 @@ class DPMSEnable(rq.Request):
 
 
 def enable(self):
-    return DPMSEnable(display=self.display,
-                      opcode=self.display.get_extension_major(extname),
-                      major_version=1,
-                      minor_version=1)
+    result = DPMSEnable(display=self.display,
+                        opcode=self.display.get_extension_major(extname),
+                        major_version=1,
+                        minor_version=1)
+
+    # Calling this function so the enabled is applied
+    info(self)
```

Comment:
See my comment in #170: this shouldn't be done by the library code. If a tool doesn't follow the usual event loop model the tool must call `Display.flush()` or `Display.sync()` as necessary to make sure requests have reached the server and been processed.